src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie

```diff
@@ -0,0 +1,88 @@
+2. Click the **Guided install** button. 
+3. Select the account you want to use for the guided install, and click **Continue**.
+4. Select **Kubernetes.**
+5. Type in a name for your cluster, and select the check box to **Gather telemetry automatically with Pixie beta**.
+
+<Callout variant="important">
+As noted in the Guided install UI, you'll get the best performance if you keep your clusters below 100 nodes. Getting up above 100 nodes can lead to excessive memory usage and scripts failing to run. And, this is a friendly reminder that autoscaling can quickly drive up your node numbers.
+</Callout>
+
+    If Pixie is already running on your cluster, expand the **Advanced options** section and select the appropriate check boxes, such as **Pixie is already running on this cluster** and **Enable Pixie auto update**.
+
+    Click **Continue**.
+
+6. Provide your Pixie API key. If you already have a key, simply paste it in the **API key** box, and click **Continue**. 
+
+If you don't have an account and existing API key, click the link to [create a Pixie account](https://work.withpixie.ai/admin/api-keys) and then on the [API key page](https://work.withpixie.ai/admin/api-keys), click **NEW KEY** to add a new key. In the **Actions** column, click the ellipsis, and then click **Copy value**. Switch back to the New Relic installation UI, paste the value in the **API key** box, and click **Continue**.
+
+When you click **Continue**, the API key is added to your New Relic account, and a deploy key is created for Pixie as well. Your Pixie account information is now stored on your New Relic account; if you follow the install steps again, you won't be asked for this information. 
+
+7. On the **Choose install method** page, copy the Helm command that's provided, and then run it on your command line. Helm installs the bundle containing the New Relic infrastructure agent, enables prometheus and Kubernetes events, and installs the Pixie integration. This takes a few minutes to complete. 
+
+![Pixie-Helm](./images/helm-pixie.png "Pixie Helm")
+
+You can run `k9s -n newrelic` to see the status of the install to the cluster. 
+
+8. Back in the Guided install, click **Continue** to open the **Listening for data** page. The install will see that data is reported to New Relic. 
+9. When you get the message, **See your data**, click **Kubernetes Cluster Explorer** to see your cluster. 
+
+Explore your cluster
+
+In the explorer, you can get a quick view of the nodes in your cluster. You can find information about each pod in he node, including CPU and memory usage. You can also find out what service is running in the pod, what the latency looks like for that service, as well as throughput and error rate. 
+
+If you want to learn more about the service, click **Explore service in Pixie**. This takes you to the Pixie UI, where you can see information about the service. <What would you do there?> 
```

Comment:
You would check whether the service is working properly, are there errors, is the latency within the expected range, etc.